src/solvers/solver_hardness.cpp

```diff
@@ -89,24 +89,28 @@ void solver_hardnesst::register_assertion_ssas(
   current_hardness = {};
 }
 
-void solver_hardnesst::register_clause(const bvt &bv)
+void solver_hardnesst::register_clause(
+  const bvt &bv,
+  const size_t solver_clause_num)
 {
   current_hardness.clauses++;
   current_hardness.literals += bv.size();
-  std::vector<int> clause;
 
   for(const auto &literal : bv)
   {
     current_hardness.variables.insert(literal.var_no());
-
-    int signed_literal = literal.var_no();
-    if(literal.sign())
-      signed_literal = -signed_literal;
-    clause.push_back(signed_literal);
   }
-  clause.push_back(0);
-  std::sort(clause.begin(), clause.end());
-  current_hardness.clause_set.push_back(clause);
+
+#ifdef DEBUG
+  std::cout << solver_clause_num << ": ";
+  for(const auto &literal : bv)
+    std::cout << literal.dimacs() << " ";
+  std::cout << "0\n";
+#endif
```

Comment:
Debug statements to output 'clause_counter_value: clause' to cross check if they match the corresponding lines in dimacs output.